### PR TITLE
feat: create an empty project

### PR DESCRIPTION
**Pull request by KurnosovaElena**

### CODE REVIEW

**Review by DmitriyG0r4anin — APPROVED** [review]

File: Management_System/Management_System/Program.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
+
+            builder.Services.AddControllers();
+            // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
```
Comment (5Ka-me) [inline Management_System/Management_System/Program.cs]: Did you leave this for yourself?

File: Management_System/Management_System/Program.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
+
+namespace Management_System
```
Comment (5Ka-me) [inline Management_System/Management_System/Program.cs]: Try no to create projects with whitespaces in names. Either change namespaces

File: Management_System/Management_System/Program.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
```
Comment (5Ka-me) [inline Management_System/Management_System/Program.cs]: Where are services? :D

File: Management_System/Management_System/Program.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            {
+                app.UseSwagger();
+                app.UseSwaggerUI();
+            }
+
+            app.UseHttpsRedirection();
+
+            app.UseAuthorization();
+
+
```
Comment (5Ka-me) [inline Management_System/Management_System/Program.cs]: In the internship  we learn good code styling.
Please get rid of blank lines

**Review by 5Ka-me — CHANGES_REQUESTED** [review]

**Review by 5Ka-me — APPROVED** [review]

**Pull request merged**